app/src/main/java/com/cris/nvh/framgiaproject/service/ServiceContract.java

```diff
@@ -0,0 +1,27 @@
+package com.cris.nvh.framgiaproject.service;
+
+public interface ServiceContract {
```

Comment:
```suggestion
public interface IService {
```
hoặc một tên bất kỳ khác. Contract có nghĩa là hợp đồng, nó có ý nghĩa ở trong MVP khi gộp Presenter và View

Reply:
vâng a :)